### DISCUSSION TITLE
Understanding billing

### DISCUSSION THREAD

**Discussion by reikdas** [community/community#26414] [Actions]
<p>Am I correct in understanding that for a public repository owned by an organization, there are unlimited free actions builds without incurring a charge?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/reikdas">@reikdas</a>,</p>
<p>Yes, for public repositories, GitHub Actions usage is free.<br>
Generally, there is no limits for minutes and storage in the public repositories.</p>
<p>More details, you can see “<a href="https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions" rel="noopener nofollow ugc"><strong>About billing for GitHub Actions</strong></a>”.</p>
